common/on_action/wc_lifestyles/wc_magic_lifestyles_on_actions.txt

```diff
@@ -0,0 +1,72 @@
+﻿magic_lifestyles_progress_pulse = { # Fires 4 times per year
```

Comment:
Done?

Reply:
You give the trait on focus picking, but don't take it back on focus changing.
My suggestion will be just doesn't give it on focus picking.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/124263510-62fdee80-db44-11eb-9c00-8372845c1c9a.png)
![image](https://user-images.githubusercontent.com/46384992/124264069-1830a680-db45-11eb-9858-ff6257764dd2.png)
![image](https://user-images.githubusercontent.com/46384992/124264097-1e268780-db45-11eb-8a19-bd5f4a4ef6db.png)

</details>

Reply:
> but don't take it back on focus changing.

If your character has publicly researched forbidden magic for whole five years, why the criminal trait should be removed just because of changed focus?

Reply:
> If your character has publicly researched forbidden magic for whole five years, why the criminal trait should be removed just because of changed focus?

How can I remove it?

Reply:
> How can I remove it?

Reset your perks - it also will reset all your lifestyle traits, including magical traits.

Reply:
> Reset your perks - it also will reset all your lifestyle traits, including magical traits.

But I have nothing to reset, I didn't spend any perk. How do you explain it to the player?
If I have experience maluses I won't be able to get one at this time.

Also, if I reset the perks but keep the focus, I get the trait back in 6 months.